docs/meshcloud.payment-methods.md

```diff
@@ -0,0 +1,94 @@
+2. A payment method's identifier is globally unique. This means you cannot create a payment method with the same identifier for multiple meshCustomers. Make sure to use an identifier that will not collide in other meshCustomers.
+
+### Creating a payment method via the meshPanel
+
+One way of creating payment methods is via the meshPanel. To do so, make sure that you have an account with the 'Partner Admin' role. Navigate to the Admin area and follow these steps:
+
+1. Click on 'Customers' on the left.
+2. Find a customer you want to create a payment method for and click on the 'Customer Payment Methods' icon on the right.
+
+    ![List Payment Methods](assets/payment_methods/customer_list_payment_methods.png)
+
+3. Click on 'Create Payment Method' at the top right.
+4. Enter a name and identifier for the new payment method.
+5. Choose a type for the payment method. Read more here on what type to choose and what the individual differences are.
+6. (This is optional) Set a maximum amount of EUR on the payment method to indicate the remaining budget for this year.
+7. (This is optional) Set an expiration date for the payment method. This is especially useful when a budget expires, e.g. at the end of the accounting year.
+8. Additionally, you can enter custom settings to inject metadata into the payment method. This is useful when you want to enhance the payment method with organizational details like the cost center number or the business unit.
+9. Click 'Save' and your new payment method will be available to the meshCustomer it was created in!
+
+### Creating a payment method via the meshstack API
+
+As automation matters, there is also the possibility to create payment methods via the meshstack API. To find the API documentation, open your meshstack and click on 'API' at the bottom of the meshPanel. A new page will open and on the left-hand side, you should see `meshPaymentMethod` listed under `meshObject Import`. Click on it and read the documentation to understand how you can create a payment method via the API.
+
+## The Payment Method Lifecycle: Assigning to Projects
+
+Now that the Partner Admin has created one or more payment methods for a meshCustomer, we are ready to link a payment method to one or more projects.
+
+First, check if the payment method is correctly created in the meshCustomer. You can do so by navigating to the meshCustomer Account area. In order to do that you need to have Customer Admin rights within the respective customer. On the left-hand side, there should be an item called 'Payment Methods' which you can navigate to. All payment methods that are created and assigned to your current meshCustomer are shown here.
+
+The payment methods that are assigned can be used for both existing projects and new projects.
+
+### Applying a payment method to an existing project
```

Comment:
It confuses me that sometimes `payment method` is written with capital letters and sometimes not.

Reply:
I'll capitalize it in each heading.